source/geod24/LocalRest.d

```diff
@@ -913,7 +912,6 @@ public final class RemoteAPI (API) : API
             C.send(this.childTid, FilterAPI(mangled, pretty));
         }
 
-
```

Comment:
Stylistic changes in their own commit, although this line seems like an arbitrary change

Reply:
It was changed from `struct` to `class`. How would you describe it?

Reply:
The comment is on a line which is removed in the middle of the file.